src/assets/translations/main.json

```diff
@@ -50,8 +50,7 @@
     "apy": "%{apy}% APY",
     "settings": "Settings",
     "legalDated": "Dated: January 2022",
-    "approveOnWallet": "Approve On Wallet",
-    "confirmOnWallet": "Confirm On Wallet"
```

Comment:
Something seems off here - we are removing these translations, which is fine - but I count 4 instances of `confirmOnWallet` in the codebase, but only 2 instances where we updated to `common.confirm`.

This PR will break those non-updated instances.

Reply:
Whoops, search was on Regex mode when I scanned the codebase.